Java/Sliding Window/SlidingWindow.java

```diff
@@ -0,0 +1,31 @@
+public class SlidingWindow {
```

Comment:
Implement all problems in Sliding Window problems in Sliding Window Folder for **bonus score**

Reply:
@mvpamansingh Please implement all the problems of Sliding Window as mentioned in issue